bin/Main.purs

```diff
@@ -212,18 +214,10 @@ main = launchAff_ do
 
           results <-
             if Array.length filesWithOptions > numThreads * 2 then do
-              -- Worker location for production bin.
+              -- Worker location
               let bundleLocation = Path.concat [ srcLocation, "bundle", "Bin.Worker", "index.js" ]
-              -- Worker location for dev.
-              let outputLocation = Path.concat [ srcLocation, "output", "Bin.Worker", "index.js" ]
               worker <-
-                oneOf
-                  [ FS.stat bundleLocation $> Worker.unsafeWorkerFromPath bundleLocation
-                  , FS.stat outputLocation $> Worker.unsafeWorkerFromPathAndExport
-                      { filePath: outputLocation
-                      , export: "main"
-                      }
-                  ]
+                FS.stat bundleLocation $> Worker.unsafeWorkerFromPath bundleLocation
```

Comment:
You're now expected to always bundle before running the local cli.

Reply:
I think we should keep this logic, but add an additional JS entry point that invokes the worker from output. This is useful for me during dev.

Reply:
Done!